src/main/java/frc/robot/commands/intake/RunIntake.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Intake;
+
+public class RunIntake extends Command {
+  Intake m_intake;
+  
+  /** Creates a new RunIntake. */
+  public RunIntake(Intake intake) {
+    m_intake = intake;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(m_intake);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    m_intake.setSpeed(0.5);
```

Comment:
If we're not using a specific value we should pass in the speed as a parameter. This will allow for quick switching of the speed between code deployments. Otherwise, great work!